pkg/util/events.go

```diff
@@ -0,0 +1,51 @@
+package util
+
+import (
+	"time"
+
+	svcsdk "github.com/aws/aws-sdk-go/service/memorydb"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	svcapitypes "github.com/aws-controllers-k8s/memorydb-controller/apis/v1alpha1"
+)
+
+var (
+	// eventsDuration represents the maximum time range worth of
+	//events to retrieve.
+	eventsDuration = time.Duration(336) * time.Hour
+	// MaxEvents represents the maximum events can to retrieve.
+	MaxEvents = int64(20)
+)
+
+func NewDescribeEventsInput(
+	sourceName *string,
+	sourceType string,
```

Comment:
Why take one string and one string pointer

Reply:
They match the parameters in `provideEvents` function in which it calls this `NewDescribeEventsInput` function.

Reply:
Should I keep all parameters in `NewDescribeEventsInput` as string?

Reply:
Fixed.

Reply:
Sorry, what's the rationale here for having a string and a string pointer?  It seems much cleaner to adopt one convention for both parameters and to adjust the places this method is called accordingly.